ts/editable/mathjax-element.ts

```diff
@@ -20,7 +20,7 @@ const mathjaxInlineDelimiterPattern = /\\\((.*?)\\\)/gsu;
  * be first translated to entities.
  */
 function translateEntitiesToMathjax(value: string) {
-    return value.replace(/&lt;/g, "{\\lt}").replace(/&gt;/g, "{\\gt}");
+    return value; // value.replace(/&lt;/g, "{\\lt}").replace(/&gt;/g, "{\\gt}");
```

Comment:
Since this function now appears to be a no-op, would it be worth removing? I'll do that now on the assumption the answer is yes (so I can get a new beta build out today), but if my assumption is wrong please feel free to add it back :-)